### PR TITLE
Master https social app awa utm campaigns again qmo

### PR DESCRIPTION
**Pull request by qmo-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by qmo-odoo** [comment]
@awa-odoo

File: addons/mass_mailing/models/utm.py [hunk 1]
```diff
@@ -28,14 +30,18 @@ class UtmCampaign(models.Model):
     replied_ratio = fields.Integer(compute="_compute_statistics", string='Replied Ratio')
     bounced_ratio = fields.Integer(compute="_compute_statistics", string='Bounced Ratio')
 
+    def _compute_mailings_count(self):
+        for campaign in self:
+            campaign.mailings_count = len(campaign.mailing_ids)
+
     @api.depends('mailing_items', 'mailing_clicked')
     def _compute_items_total(self):
+        super(UtmCampaign, self)._compute_items_total
```
Comment (awa-odoo) [inline addons/mass_mailing/models/utm.py]: Missing "()"

Leads me to wonder if you have tested mailing + social and check that stats are correctly summed.
Might be worth to check...

File: addons/utm/views/utm_campaign_views.xml [hunk 2]
```diff
@@ -23,6 +23,9 @@
         <field name="model">utm.campaign</field>
         <field name="arch" type="xml">
             <form string="UTM Campaign">
+                <header style="min-height:31px;">
```
Comment (awa-odoo) [inline addons/utm/views/utm_campaign_views.xml]: Inline-style :(
Also why?

**Review by awa-odoo — CHANGES_REQUESTED** [review]
To check before cherry-picking

**Comment by awa-odoo** [comment]
Merged in another PR

**Pull request closed without merging**